roadmap/index.md

```diff
@@ -8,7 +8,5 @@ In an effort to be transparent with our roadmap, we are publishing in advance th
 
 ## Upcoming changes
 
-### 2015
-
-
-#### ”Why am I seeing this?”
+We don't have any upcoming plans really, since this is the enabling infrastructure and it's doing it's job really well. We are rather focusing on improving our
```

Comment:
I really think it should be `We are rather focused on improving our etc. etc` instead of `rather focusing`.